DataDefinitions/Properties/Modules.fr.resx

```diff
@@ -334,10 +353,6 @@
     <value>Lance mine électrique</value>
     <comment>Module ShockMineLauncher</comment>
   </data>
-  <data name="ShutdownFieldNeutraliser" xml:space="preserve">
```

Comment:
Why are we removing this?

Reply:
Because it's English text that was pasted into the French resource by mistake. We need the French translation.